### PR TITLE
console errors delated

### PR DESCRIPTION
**Pull request by igi144**

### CODE REVIEW

File: src/common/MovieContainer/index.js
Comment (WiktoriaNizielska) [inline src/common/MovieContainer/index.js:1]: I don't understand why useParams are in this file. Common folder has style files.

File: src/common/MovieContainer/index.js [hunk 1]
```diff
@@ -12,42 +13,51 @@ import {
     Star
 } from "./styled"
 
+
+
```
Comment (WiktoriaNizielska) [inline src/common/MovieContainer/index.js:17]: empty lines to delete

File: src/core/App.js [hunk 2]
```diff
@@ -35,16 +36,16 @@ function App() {
...
       </StyledNavigation>
       <Switch>
-        <Route path="/movies">
-          <MovieList />
-        </Route>
-        <Route path="/movie">
+        <Route path="/movie/:id">
           <MovieContainer />
         </Route>
+        <Route path="/movies/">
```
Comment (WiktoriaNizielska) [inline src/core/App.js:42]: it shouldn't be "/movies"? without "/" at the end?

File: src/core/App.js [hunk 3]
```diff
@@ -35,16 +36,16 @@ function App() {
...
+          <MovieList />
+        </Route>
         <Route path="/people">
         </Route>
         <Route path="/">
-          <Redirect to="/movies" />
+          <Redirect to="/movies/" />
```
Comment (WiktoriaNizielska) [inline src/core/App.js:48]: Here the same situation?

File: src/features/movies/movieList/index.js [hunk 4]
```diff
@@ -46,11 +51,15 @@ export const MovieList = () => {
             />
             <TextWrapper>
               <Container>
-                <Title>{movie.original_title}</Title>
+                <Link to={`/movie/${movie.id}`}>
+                  <Title>
```
Comment (WiktoriaNizielska) [inline src/features/movies/movieList/index.js:55]: Title has an underline, it has to be changed in styled

**Review by WiktoriaNizielska — COMMENTED** [review]
Great job!
One comment ony.
When you click in the title, star and tags show up. So its next thing to make :)

moveid

File: src/common/MovieContainer/index.js
Comment (igi144) [reply src/common/MovieContainer/index.js:1]: It was only for the test, I forgot to delete that  :)

**Review by WiktoriaNizielska — APPROVED** [review]
Nicely done corrections!

**Pull request merged**